### ISSUE TITLE
[ui5-carousel]: carousel heading content not announced by screen reader

### ISSUE THREAD

**Issue by caroline-ferri** [SAP/ui5-webcomponents#11399]
### Bug Description

Content within the carousel is not being announced when the carousel item is tabbed to. 

### Affected Component

ui5-carousel

### Expected Behaviour

The content within the carousel like heading items, buttons, etc. should be announced by the JAWS screen reader. With version 2.8.0 the carousel content is announced, upon upgrading to 2.9.0 the carousel content is not being announced. 

### Isolated Example

https://codesandbox.io/p/sandbox/laughing-browser-l72jfg?file=%2Fsrc%2FApp.js%3A18%2C18

### Steps to Reproduce

1. Open https://codesandbox.io/p/sandbox/laughing-browser-l72jfg?file=%2Fsrc%2FApp.js%3A18%2C18 
2. Start JAWS 2025 and navigate to sample
3. Tab to carousel item. Note that the carousel item content/aria-labels are not announced. 
4. Open https://codesandbox.io/p/devbox/intelligent-bhabha-wg4j7f?file=%2Fsrc%2FApp.js%3A16%2C28 (same code sample but using 2.8.0)
5. Start JAWS and navigate to sample
6. tab to carousel item. Note that the carousel item content/aria-labels are announced. 
...

### Log Output, Stack Trace or Screenshots

<img width="1639" alt="Image" src="https://github.com/user-attachments/assets/b72f390c-a69f-4b72-befe-d0aa0da5bda6" />

### Priority

Medium

### UI5 Web Components Version

2.9.0

### Browser

Chrome

### Operating System

MacOS

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by TeodorTaushanov** [comment]
Hello,

In the nightly version (2.16.0-rc.3) of UI5 Web Components we implemented the necessary accessibility specification:
https://ui5.github.io/webcomponents/nightly/components/Carousel/

Please note that not every element on the carousel page needs to be read when the page is focused; this behavior depends on the screen readers.

Best,
Teodor
